### ISSUE TITLE
Add a Readme section for Bazelisk users

### ISSUE THREAD

**Issue by dzintars** [bazelbuild/vim-bazel#17]
Hi, all. Thank you for this plugin.
Today came across issue that Bazel binary cant be found by this plugin.
That was because i was using plain `alias bazel='bazelisk'` in my `~/.zshrc`.
The fix of this was just a simple `sudo ln -s /usr/local/bin/bazelisk /usr/local/bin/bazel`. I think this could augment the readme of this plugin pretty well.

**Comment by dzintars** [comment]
Alternative is just to rename `bazelisk` binary to `bazel`, but... it's less explicit.

**Comment by dbarnett** [comment]
Yeah, symlink looks like a decent workaround, and the plugin could have an explicit way to configure the path to the bazel executable. Note I recently added 7d816323aa for a similar override at command invocation time, but overriding at that level is probably not well-suited to the bazelisk case.

Would you mind suggesting some phrasing of the explanations you'd want to see in the README and/or vim help?
